spyder/config/user.py

```diff
@@ -13,14 +14,14 @@
 from __future__ import print_function
 
 # Std imports
+import ast
 import os
 import re
 import os.path as osp
 import shutil
 import time
 
 # Local imports
-from spyder import __version__

```

Comment:
> Are you sure this removal is ok?

Well, I searched for `__version__` and I checked that Spyder still runs and the tests pass, both with Python 2 and Python 3. But it may still be used in external code, so no, I'm not 100% sure. But I feel I'm sure enough.